python/cudf/cudf/_lib/sort.pyx

```diff
@@ -221,7 +225,9 @@ def rank_columns(Table source_table, object method, str na_option,
     """
     Compute numerical data ranks (1 through n) of each column in the dataframe
     """
-    cdef table_view source_table_view = source_table.data_view()
+    cdef table_view source_table_view = table_view_from_table(
+        source_table, True
+    )
```

Comment:
From readability perspective, I think we should explicitly use keyword argument when there's a lone boolean value:
```suggestion
    cdef table_view source_table_view = table_view_from_table(
        source_table, ignore_index=True
    )
```